Add introduction, ignore/remove generated file

ericzolf opened the pull request:
* move some of the introduction from the roles' readme to the main one
* complete it with a few generic words about purpose and content of the
  repo
* ignore all generated files (html or pdf) and remove existing one.

acozine commented on roles/README.adoc:
```diff
@@ -1,17 +1,9 @@
+= Roles Good Practices
 
 This document is intended to be a standards document for the Ansible roles contained in the
 AGP Roles repositories.
```
To match the main page above, which abbreviates Good Practices for Ansible as GPA:
```suggestion
GPA Roles repositories.
```

ericzolf replied:
I removed it altogether, the introduction anyway didn't fit the context.

acozine reviewed (commented):
A couple of small wordsmithing suggestions.

ericzolf commented:
I've addressed the comments from @acozine but she's not member of the team so I need someone from @redhat-cop/automation-cop-members to approve.
Alicia, if you want to get write access, you need first to become member of redhat-cop (I think you need to request it and it'll be granted, if not let me know).

sean-m-sullivan reviewed (approved):
I think these look like good changes. Propose at least one more review before merge though.

ericzolf commented:
We'll wait until Wednesday and validate the merge in the CoP meeting.

jscar-hawk commented:
I'll update and take note of what the Wednesday Meeting decides.
